src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -617,6 +617,13 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->then(function ($v) { return (string) $v; })
                             ->end()
                         ->end()
+                        ->arrayNode('mappings')
+                            ->prototype('array')
+                                ->children()
+                                    ->scalarNode('file')->end()
```

Comment:
in preparation for the "dir" key to include all mapping files automatically from this directory.